.eslintrc.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
   ],
   rules: {
     /* general */
-    'arrow-parens': [1, 'as-needed'],
+    'arrow-parens': [1, 'always'],
```

Comment:
it looks like this change is responsible for the most styling noise in the PR, what was the motivation?

Reply:
The responsible is prettier v2. I rollbacked it’s upgrade and git back to the initial configuration.